### DISCUSSION TITLE
Tutorial: Setting up self hosted RustDesk server on Windows

### DISCUSSION THREAD

**Discussion by pedjas** [rustdesk/rustdesk#2152] [Show and tell]
After some struggle to set self hosted RustDEsk server and make everythng work I wrote down notes for myself so if I need to do similar setting again, I do not have to deal with the same issues.

Tutorial covers installation on Windows behind NAT with clients both on Internet and local network, which is probably most often needed setup.

https://pedja.supurovic.net/setting-up-self-hosted-rustdesk-server-on-windows/?lang=lat

I am open to all suggestions if tutorial needs improvements and corrections.

**Comment by rustdesk** [comment]
Thank you
